package.json

```diff
@@ -91,17 +91,17 @@
   },
   "devDependencies": {
     "chai": "~3.5.0",
-    "chai-as-promised": "^5.3.0",
+    "chai-as-promised": "^6.0.0",
```

Comment:
Looks like we use chai-as-promised in one test for its `isRejected` function here: https://github.com/screwdriver-cd/screwdriver/blob/master/test/plugins/auth.test.js#L108